tiflash/tiflash-configuration.md

```diff
@@ -208,19 +212,25 @@ delta_index_cache_size = 0
 [server]
     engine-addr = The external access address of the TiFlash coprocessor service.
 [raftstore]
-    ## Specifies the number of threads that handle snapshots.
+    ## The allowable number of threads in the pool that flushes Raft data to storage.
+    apply-pool-size = 4
+
+    ## The allowable number of threads that process Raft, which is the size of the Raftstore thread pool.
+    store-pool-size = 4
+
+    ## The number of threads that handle snapshots.
     ## The default number is 2.
     ## If you set it to 0, the multi-thread optimization is disabled.
     snap-handle-pool-size = 2
 
-    ## Specifies the shortest interval at which Raft store persists WAL.
+    ## The shortest interval at which Raft store persists WAL.
     ## You can properly increase the latency to reduce IOPS usage.
     ## The default value is "4ms".
     ## If you set it to 0ms, the optimization is disabled.
     store-batch-retry-recv-timeout = "4ms"
 [security]
     ## New in v5.0. This configuration item enables or disables log redaction.
-    ## If the configuration value is set to true,
+    ## If the configuration value is set to "true",
     ## all user data in the log will be replaced by ?. The default value is false.
     redact-info-log = false
```

Comment:
"redact-info-log" is not in the Chinese doc. Please check if it should be added.

Reply:
will fix in another PR.